### DISCUSSION TITLE
Join us on Discord

### DISCUSSION THREAD

**Discussion by jxnl** [instructor-ai/instructor-js#48] [Announcements]
https://discord.gg/CV8sPM5k5Y

Against better judgment there so much stuff going on, I have to make a discord. Please join! Also would love help with moderation.

**Comment by harlantwood** [comment]
@jxnl I get this when I try to join:

<img width="836" alt="Screenshot 2024-06-21 at 10 47 37 PM" src="https://github.com/instructor-ai/instructor-js/assets/38769/18457508-71b4-4fa5-be86-37cbe77a38d3">
